examples/config-example.tmpl

```diff
@@ -0,0 +1,54 @@
+issuer: {{ getenv "DEX_ISSUER" "http://127.0.0.1:5556/dex" }}
+
+storage:
+  type: sqlite3
+  config:
+    file: {{ getenv "DEX_STORAGE_SQLITE3_CONFIG_FILE" "/etc/dex/dex.db" }}
+
+web:
+{{- if getenv "DEX_WEB_HTTPS" "" }}
+  https: {{ .Env.DEX_WEB_HTTPS }}
+  {{- if getenv "DEX_WEB_TLS_KEY" }}
+  tlsKey: {{ .Env.DEX_WEB_TLS_KEY }}
+  {{- else }}
+{{- fail "$DEX_WEB_TLS_KEY is required" }}
+  {{- end }}
+  {{- if getenv "DEX_WEB_TLS_CERT" "" }}
+  tlsCert: {{ .Env.DEX_WEB_TLS_CERT }}
+  {{- else }}
+{{- fail "$DEX_WEB_TLS_CERT is required" }}
+  {{- end }}
+{{- end }}
+  http: {{ getenv "DEX_WEB_HTTP" "0.0.0.0:5556" }}
+
+{{- if getenv "DEX_TELEMETRY_HTTP" "" }}
+telemetry:
+  http: {{ .Env.DEX_TELEMETRY_HTTP }}
+{{- end }}
+
+expiry:
+  deviceRequests: {{ getenv "DEX_EXPIRY_DEVICE_REQUESTS" "5m" }}
+  signingKeys: {{ getenv "DEX_EXPIRY_SIGNING_KEYS" "6h" }}
+  idTokens: {{ getenv "DEX_EXPIRY_ID_TOKENS" "24h" }}
+  authRequests: {{ getenv "DEX_EXPIRY_AUTH_REQUESTS" "24h" }}
+
+logger:
+  level: {{ getenv "DEX_LOGGER_LEVEL" "info" }}
+  format: {{ getenv "DEX_LOGGER_FORMAT" "text" }}
+
+oauth2:
+  responseTypes: {{ getenv "DEX_OAUTH2_RESPONSE_TYPES" "[code]" }}
+  skipApprovalScreen: {{ getenv "DEX_OAUTH2_SKIP_APPROVAL_SCREEN" "false" }}
+  alwaysShowLoginScreen: {{ getenv "DEX_OAUTH2_ALWAYS_SHOW_LOGIN_SCREEN" "false" }}
+{{- if getenv "DEX_OAUTH2_PASSWORD_CONNECTOR" "" }}
+  passwordConnector: {{ .Env.DEX_OAUTH2_PASSWORD_CONNECTOR }}
+{{- end }}
+
+enablePasswordDB: {{ getenv "DEX_ENABLE_PASSWORD_DB" "true" }}
+
+connectors:
```

Comment:
Yeah, firstly, I thought the same way. However, it works :)

Reply:
Maybe it will be better to add one more variable. Let's call it `DEX_CONNECTORS_ADDITIONAL` for now. 
Users will be able to specify custom connectors via it. Example:

Template:
```
connectors:
{{- if getenv "DEX_CONNECTORS_ENABLE_MOCK" }}
- type: mockCallback
  id: mock
  name: Example
{{- end }}
{{ getenv "DEX_CONNECTORS_ADDITIONAL" | jsonArray | toYAML }}
```
ENV:
```
DEX_CONNECTORS_ENABLE_MOCK=yes
DEX_CONNECTORS_ADDITIONAL='[{"type":"mockCallback","id":"secondMock","name":"Example2"}]'
```
Result:
```yaml
connectors:
- type: mockCallback
  id: mock
  name: Example
- id: secondMock
  name: Example2
  type: mockCallback
```